tests/helpers.py

```diff
@@ -19,7 +19,7 @@
 def create_unique_resource_name(resource_description, resource_type):
     """
     Creates a unique object name by using the object_description
-    and object_type, as well as the current date/time string.
+    and object_type, and a random number between 100 to 1000000
```

Comment:
random number is not between 100 and 1000000 as you compose from 2 numbers from random range  100 - 1000000 . and  1000 - 10000 . And then you multiply them so lowest possible is 99 * 999 = 98901. I am proposing some diff approach, se my comment below.

Reply:
Lowest possible would be 100*1000,  I can update the comment which I remembered later, but having high number is digits is required to avoid collision for scale tests.